### ISSUE TITLE
Can't upload same file twice in a row

### ISSUE THREAD

**Issue by noahwagner04** [AutomatonBuilderGUI/AutomatonBuilderGUI#125]
When trying to upload a file twice in a row, no automata loads the second time.

Reproduction Steps:
1. Put together a simple automaton
2. Download the automaton JSON file
3. Optionally delete the current automaton (makes it easier to see the bug)
4. Upload that same automaton file
5. Optionally delete the current automaton (makes it easier to see the bug)
6. Upload that same automaton file

At this point there should be no new automaton that is loaded. Note that the only necessary steps are 1, 2, 4, and 6. Clearing the current automaton just makes it easier to see that no automaton loads on step 6.

This bug seems to only exclusively happen with loading the same exact file twice in a row. If you load another automaton file in-between the other two uploads, everything seems to work.

**Comment by obimax99** [comment]
I'll take this!

**Comment by Meorge** [comment]
At our meeting today, we discussed the possibility that this issue had been resolved by #127. However, I just tried following the reproduction steps and they still seem to be resulting in no automaton for me.

The following steps also cause me to get an empty automaton:
1. Open the program (load or refresh page).
2. Upload an automaton file. The automaton should appear as expected.
3. Click the "Clear the Automaton" button. The automaton should disappear.
4. Upload the same automaton file as in Step 2. No automaton will appear.
